src/consts.ts

```diff
@@ -62,9 +62,9 @@ export const MaxPayoutDays = 60;
 export const EndpointPrice = 'https://api.binance.com/api/v3';
 export const ApiEndpoints = {
   priceChange: `${EndpointPrice}/ticker/24hr?symbol=`,
-  subscanRewardSlash: '/api/plugin/staking/rewardsSlashes',
-  subscanPoolRewards: '/api/plugin/staking/poolRewards',
-  subscanEraStat: '/api/plugin/staking/eraStat',
+  subscanRewardSlash: '/api/v2/scan/account/reward_slash',
+  subscanPoolRewards: '/api/scan/nomination_pool/rewards',
+  subscanEraStat: '/api/scan/staking/era_stat',
```

Comment:
@nbass3 I understand these changes are needed because of the switch to using Subscan as a hosted service. Indeed we've adjusted these API endpoints ourselves in the beginning and now you are just reverting them back. 

Question: are we using only Subscan as a service or using both Subscan as a service and self-hosted Subscan fork at the same time together ? (See my comments for `src/config/networks.ts` for some more context). If both are used at the same time that's going to fail because API paths are different on the self-hosted Subscan fork.